pkg/kubectl/cmd/BUILD

```diff
@@ -1,3 +1,5 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I think the problem is when I execute `./hack/update-bazel` it will auto generated those files for me, but visibility is wrong, could we auto generate the right one ? so I don't have to manually edit those files.